### DISCUSSION TITLE
Installed Docker but where are executables?

### DISCUSSION THREAD

**Discussion by dradamskelton** [deepmodeling/deepmd-kit#14] [Q&A] [answered]
I got the docker to install apparently successfully on Ubuntu
i.e. it gave the following

Successfully built 7ed8bb3adece
Successfully tagged deepmd-kit_docker:latest

I do not, however, know where the executables are.  I have tried locating dp_frz  dp_ipi  dp_test  dp_train files but to no avail.

I want start again but don't know how to "clean" the process.

Thanks

Adam

**Comment by frankhan91** [comment, marked as answer]
The message you give indicates that you have built the Dockerfile successfully. To get access to the executables, you should first derive a container from the image  --  an example command is like "docker run -t -i deepmd-kit_docker". Then you can follow the instructions of [DeePMD-kit Manual](https://github.com/deepmodeling/deepmd-kit#train-a-model) to find the executables, train a model, an so on.
